docs/UserGuide.md

```diff
@@ -35,100 +39,194 @@
+
+>If you want to view any of the Todo tasks, Scheduled tasks or Completed tasks only, then you only<br> need to type in �todo�, �schedule� or �completed� as the keyword after the command list. <br>
+> <img src="images/UI_Prototype/WhatNowUI-ListCompleted-Colour.png" width="600">
+
+
+#### Deleting a task : `delete`
+Deletes a task from WhatNow. Reversible.<br>
+Format: `delete todo/schedule/completed INDEX`
+
+> * Deletes the task at the specified `INDEX` from todo tasks, schedule or completed tasks as specified. <br>
+> * The index refers to the index number shown in the current listing.<br>
+> * The index **must be a positive integer** 1, 2, 3, ...
 
 Examples: 
-* `add John Doe p/98765432 e/johnd@gmail.com a/John street, block 123, #01-01`
-* `add Betsy Crowe p/1234567 e/betsycrowe@gmail.com a/Newgate Prison t/criminal t/friend`
+* `delete todo 4`<br>
+  Deletes the 4th todo task from the displayed list in WhatNow. <br>
+  <img src="images/UI_Prototype/WhatNowUI-DeleteTodo-Colour.png" width="600">
+* `delete schedule 2`<br>
+  Deletes the 2nd schedule task from the displayed list in WhatNow. <br>
+  <img src="images/UI_Prototype/WhatNowUI-DeleteSchedule-Colour.png" width="600">
 
-#### Listing all persons : `list`
-Shows a list of all persons in the address book.<br>
-Format: `list`
 
-#### Finding all persons containing any keyword in their name: `find`
-Finds persons whose names contain any of the given keywords.<br>
-Format: `find KEYWORD [MORE_KEYWORDS]`
+#### Updating a task : `update`
+Updates a task from the list displayed<br>
+Format: `update todo/schedule INDEX description/date/start/end/status/priority  NEW_VALUE`
 
-> * The search is case sensitive. e.g `hans` will not match `Hans`
-> * The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
-> * Only the name is searched.
-> * Only full words will be matched e.g. `Han` will not match `Hans`
-> * Persons matching at least one keyword will be returned (i.e. `OR` search).
-    e.g. `Hans` will match `Hans Bo`
+> * Updates the description/date/start/end/status/priority of the task at the specified `INDEX`<br> from todo tasks or schedule as specified.<br> 
+> * The index refers to the index number shown in the current listing.<br>
+> * The index **must be a positive integer** 1, 2, 3, ...
 
 Examples: 
-* `find John`<br>
-  Returns `John Doe` but not `john`
-* `find Betsy Tim John`<br>
-  Returns Any person having names `Betsy`, `Tim`, or `John`
+* `update todo 3 description Avengers`<br>
+   Selects the 3rd todo task from the displayed list and changes the task description to �Avengers�
+* `update todo 4 date 11 Nov`<br>
+   Selects the 4th todo task from the displayed list and changes the type of task from todo to scheduled. 
+* `update schedule 5 time 8:30pm`<br>
+   Selects the 5th schedule task from the displayed list and changes the time to 8:30pm. <br>
+<img src="images/UI_Prototype/WhatNowUI-UpdateSchedule-Colour.png" width="600"> 
```

Comment:
Yup, it will always highlight the latest changes. Okay thanks for the reminder! We will mention it :)